site/docs/operations/gc-related-concerns.md

```diff
@@ -0,0 +1,118 @@
+---
+title: Concerns Related to Garbage Collection
+description: Choosing the right JDK-GC combination is important.
+---
+
+## Choice of JDK and GC
+
+In May 2020 we did extensive testing of Hazelcast Jet with several
+combinations of JDK and garbage collector. We tried the following JDK/GC
+combinations:
+
+1. JDK 8 with the default Parallel collector and the optional
+   ConcurrentMarkSweep and G1
+2. JDK 11 with the default G1 collector
+3. JDK 14 with the default G1 as well as the experimental Z and
+  Shenandoah
+
+These are our key findings:
+
+1. Due to the large improvements in the later versions, JDK version 8
```

Comment:
```suggestion
1. Due to the large improvements in the latter versions, JDK version 8
```

Reply:
"Later" is appropriate, "latter" implies a former-latter distinction which we aren't making here.

Reply:
I'm not sure. I thought "latter" here means "recent". See definitions 2 and 3 [here](https://www.dictionary.com/browse/latter). When "later" is used, it's later than what? Maybe it could be "late versions".

Reply:
Versions later than 8. "Latter" could possibly still mean something, like the versions I mention above, after 8. But saying that wasn't my intention. "Later" conveys exactly the meaning I intended.